charts/container-startup-autoscaler/CHANGELOG.md

```diff
@@ -2,14 +2,14 @@
 - Based on [Keep a Changelog](https://keepachangelog.com/en/1.1.0/).
 - This project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
-## 1.0.0
```

Comment:
We could use a [GitHub Action for semantic releases](https://github.com/cycjimmy/semantic-release-action) so we don't have to manually update this file. [Example](https://github.com/ExpediaGroup/insights-explorer/blob/main/.github/workflows/release.yaml) from another project and [what](https://github.com/ExpediaGroup/insights-explorer/blob/main/CHANGELOG.md) it would generate when following [conventional commits](https://www.conventionalcommits.org/en/v1.0.0/).

Reply:
Yeah, was thinking this as I was moving stuff around. Care to create an issue and we can address later?

Reply:
#6